### DISCUSSION TITLE
Set Icon image based on value in HA (i.e. Spofity album cover)

### DISCUSSION THREAD

**Discussion by bgnzo818** [cgiesche/streamdeck-homeassistant#205] [Feature Requests]
I love this plugin and it keeps getting better. When using a media_player domain, is it possible to use the {{entity_picture}} as the icon? I've seen this done through the Spotify and Sonos plugs but would prefer using this plugin as neither offer dial options.

**Comment by CommanderB** [comment]
Upvote for this. I have a workarround using the WS Proxy SD plug in and sending the image from a Node Red flow.  But native support in this plug in would be excellent.

**Comment by cgiesche** [comment]
Feel free to try out an early development version. Not yet tested on windows - may work - may not work.
Backup your profile in case anything goes wrong :)

https://github.com/cgiesche/streamdeck-homeassistant/releases/tag/3.8.0-alpha.3
